source/material_model/diffusion_dislocation.cc

```diff
@@ -125,7 +125,7 @@ namespace aspect
       // Otherwise, calculate the square-root of the norm of the second invariant of the deviatoric-
       // strain rate (often simplified as epsilondot_ii)
       const double edot_ii = std::max(std::sqrt(std::fabs(second_invariant(deviator(strain_rate)))),
-                                      min_strain_rate * min_strain_rate);
+                                      min_strain_rate);
```

Comment:
Yes, I think so. This should be the strain rate, not the strain rate squared. 
The purpose of this line is only to provide a minimum strain rate in case the strain rate is low, so in the old version this value was just much smaller than intended (the default value is something like 1e-20). So this did not lead to wrong results, it just made the model crash more often.